doc/toolbox.1.md

```diff
@@ -90,35 +90,35 @@ Same as `--log-level=debug`. Use `-vv` to include `--log-podman`.
 
 Commands for working with toolbox containers and images:
 
-**toolbox-create(1)**
+**toolbox create(1)**
```

Comment:
The thing is that `toolbox-create(1)` is pointing to the *manual* for the `toolbox create` command, and not the command itself.  Those parentheses indicate the section within the entire collection of manuals where the `toolbox-create` manual page exists.

It's a convention that's mentioned in [man(1)](https://man7.org/linux/man-pages/man1/man.1.html), and is used in `podman(1)` and [git(1)](https://git-scm.com/docs/git) to list the manuals for each command.

If you want to provide some very brief example of how to use Toolbx, then we can replicate the *Usage* section from the [website](https://containertoolbx.org/install/).